src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -1012,6 +1013,31 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
         self ! InitialStartCall(callKey, CallActor.Start)
     }
 
+    /**
+      * If using Mac OS X with Docker Machine, by default Cromwell can only use the -v flag to 'docker run'
+      * if running from somewhere within the user's home directory.  If not, then -v will mount to the
+      * virtual machine that Docker Machine creates.  This ends up bubbling up to Cromwell when it can't find
+      * an RC file in the call's output directory (because it's actually on the virtual machine).
+      *
+      * This function will at least detect this case and log some WARN messages.
+      */
+    def detectDockerMachinePossibleMisusage(backendCall: LocalFileSystemBackendCall, logger: WorkflowLogger) = {
+      val os = Option(System.getProperty("os.name"))
+      val homeDir = Option(System.getProperty("user.home")).map(Paths.get(_).toAbsolutePath.toString)
+      val callRoot = backendCall.callRootPath.toAbsolutePath.toString
+      val dockerMachineName = sys.env.get("DOCKER_MACHINE_NAME")
+      if (dockerMachineName.nonEmpty && os.contains("Mac OS X") && homeDir.isDefined && !callRoot.startsWith(homeDir.get)) {
+        logger.warn(s"You are running Docker using Docker Machine and your working directory is not a subdirectory of ${homeDir.get}")
+        logger.warn(s"By default, Docker Machine only shares your home directory, which means Cromwell can only be run from under your home directory")
+        logger.warn(s"See https://docs.docker.com/engine/userguide/dockervolumes/ for more information")
+      }
+    }
+
+    backendCall match {
```

Comment:
Scott and I discussed this and agreed the logic should be moved to Main and not involve the local backend at all.  i.e. this would warn for the combination of Mac + DOCKER_MACHINE_NAME + cromwell PWD not under user home directory.  This allows the error to be printed once at startup and futureproofs for dynamically selectable backends.